### DISCUSSION TITLE
Is it any way, how to change the height of bottom waveform?

### DISCUSSION THREAD

**Discussion by SolarGuy8** [katspaugh/wavesurfer.js#3493] [Q&A] [answered]
Is it any way, how to change the height only for bottom waveform?

<img width="857" alt="Screenshot 2024-01-16 at 16 15 28" src="https://github.com/katspaugh/wavesurfer.js/assets/36670965/b6855da4-7e05-401e-be7c-d9f58df4cd5b">

**Comment by katspaugh** [comment, marked as answer]
You can write a custom render function like in this example: https://wavesurfer.xyz/examples/?custom-render.js

**Reply by SolarGuy8** [reply]
thank you!
